ocaml/src/capabilities.ml

```diff
@@ -0,0 +1,54 @@
+(*
+Copyright (C) 2016 iNuron NV
+
+This file is part of Open vStorage Open Source Edition (OSE), as available from
+
+
+    http://www.openvstorage.org and
+    http://www.openvstorage.com.
+
+This file is free software; you can redistribute it and/or modify it
+under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+as published by the Free Software Foundation, in version 3 as it comes
+in the <LICENSE.txt> file of the Open vStorage OSE distribution.
+
+Open vStorage is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY of any kind.
+ *)
+
+
+
+module OsdCapabilities = struct
```

Comment:
Are these really osd capabilities or rather asd capabilities? (The todos in alba_osd.ml & kinetic_client.ml point towards asd capabilities?)
And is this concept actually needed?
I only expected some form of osd info to be sent back by the proxy OsdInfo call when we're talking about an actual 'rora'.
```
type rora_info = { ip : string; port : int; } (* note that port is the rora-port; maybe a few other things are needed too? *)
```

and then in the proxy protocol definition
```
| OsdInfo : (unit, osd_id * rora_info) Std.counted_list
```